### DISCUSSION TITLE
Custom spacing between items? Where and How?

### DISCUSSION THREAD

**Discussion by Fischmuetze** [jordanbaird/Ice#143] [General]
Feature list is pointing to "Custom spacing between items" but I can't find a configuration of it like in Bartender or similar. How can I control the spacing between items?

**Comment by illuminaresolutions** [comment]
It’s unchecked because it’s on the roadmap and not a current feature.

**Comment by Fischmuetze** [comment]
ok thanksAm 11.06.2024 um 17:08 schrieb Illuminare Solutions ***@***.***>:﻿
It’s unchecked because it’s on the roadmap and not a current feature.
